src/storage/src/source/source_reader_pipeline.rs

```diff
@@ -1119,22 +1132,24 @@ where
 /// Receives un-timestamped batches from the source reader and updates to the
 /// remap trace on a second input. This operator takes the remap information,
 /// reclocks incoming batches and sends them forward.
-fn reclock_operator<G, S: 'static>(
+fn reclock_operator<G, K, V, D>(
     scope: &G,
     config: RawSourceCreationConfig,
     mut timestamper: ReclockFollower,
-    batches: timely::dataflow::Stream<G, SourceMessageBatch<S::Key, S::Value, S::Diff>>,
-    remap_trace_updates: timely::dataflow::Stream<G, (PartitionId, Timestamp, MzOffset)>,
+    batches: TimelyStream<G, SourceMessageBatch<K, V, D>>,
+    remap_trace_updates: TimelyStream<G, (PartitionId, Timestamp, MzOffset)>,
 ) -> (
     (
-        Vec<timely::dataflow::Stream<G, SourceOutput<S::Key, S::Value, S::Diff>>>,
-        timely::dataflow::Stream<G, SourceError>,
+        Vec<TimelyStream<G, SourceOutput<K, V, D>>>,
```

Comment:
I'd much rather import timely's `Stream` normally, and import `tokio_stream::Stream as `AsyncStream`, as in the source pipeline I am in timely-mode when thinking about streams